src/components/base/Input/Input.tsx

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components';
+
+interface InputProps {
+  width: number;
+  height: number;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  name: string;
+  error: boolean;
+}
+
+const Input = ({
+  width,
+  height,
+  onChange,
+  name,
+  error,
+  ...props
+}: InputProps) => {
+  let status: string = '';
+
+  if (error) {
+    status = 'error';
+  }
+
+  const inputStyle: React.CSSProperties = {
+    width: width,
+    height: height,
+  };
+
+  return (
+    <InputContainer
+      width={width}
+      height={height}
+      onChange={onChange}
+      name={name}
+      className={status || undefined}
+      style={{ ...inputStyle }}
+      {...props}
+    />
+  );
+};
+
+const InputContainer = styled.input`
+  display: block;
+  box-sizing: border-box;
+  padding: 0 16px; //
```

Comment:
```suggestion
  padding: 0 16px;
```
제안사항 수정하실 거면 그 김에 이것도 수정 부탁드려요 ㅎ